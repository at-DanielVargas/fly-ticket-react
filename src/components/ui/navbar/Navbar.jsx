import React from 'react'
import { Link } from 'react-router-dom'
import { HiOutlineTicket } from 'react-icons/hi'
import { MdAirplaneTicket } from 'react-icons/md'

export const Navbar = () => {
  return (
    <div className='navbar'>
      <div className='container d-flex'>
        <div className='left'>
          <span className='icon'>
            <Link to='/'>
              <HiOutlineTicket />
              <MdAirplaneTicket className='text-primary' />
            </Link>
          </span>
          <Link to='/' className='name'>
                        Fly<span className='txt-purple'>Ticket</span>
          </Link>
          <nav>
            <Link to='/'>Reservar</Link>
            <Link to='/'>Información</Link>
            <Link to='/'>Mi reserva</Link>
          </nav>
        </div>
        <div className='right'>
          <nav>
            <Link to='/auth/login'>Iniciar sesión</Link>
            <Link to='/'>Registro</Link>
          </nav>
        </div>
      </div>
    </div>
  )
}
