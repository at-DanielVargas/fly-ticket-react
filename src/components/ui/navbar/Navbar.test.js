
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import { Navbar } from './Navbar'

describe( 'Navbar Component', () => {
  let wrapper

  beforeEach( () => {
    wrapper = shallow( <Navbar /> )
  } )

  test( 'should be render Navbar component ', () => {
    expect( wrapper ).toMatchSnapshot()
  } )
} )
