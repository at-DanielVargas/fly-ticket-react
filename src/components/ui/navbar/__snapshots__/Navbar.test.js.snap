// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component should be render Navbar component  1`] = `
<div
  className="navbar"
>
  <div
    className="container d-flex"
  >
    <div
      className="left"
    >
      <span
        className="icon"
      >
        <Link
          to="/"
        >
          <HiOutlineTicket />
          <MdAirplaneTicket
            className="text-primary"
          />
        </Link>
      </span>
      <Link
        className="name"
        to="/"
      >
        Fly
        <span
          className="txt-red"
        >
          Ticket
        </span>
      </Link>
      <nav>
        <Link
          to="/"
        >
          Reservar
        </Link>
        <Link
          to="/"
        >
          Información
        </Link>
        <Link
          to="/"
        >
          Mi reserva
        </Link>
      </nav>
    </div>
    <div
      className="right"
    >
      <nav>
        <Link
          to="/auth/login"
        >
          Iniciar sesión
        </Link>
        <Link
          to="/"
        >
          Registro
        </Link>
      </nav>
    </div>
  </div>
</div>
`;
